src/fairseq2/recipes/hg/asr_eval.py

```diff
@@ -232,6 +271,79 @@ def load_wav2vec2_asr_evaluator(
         gang=gang,
         data_reader=pipeline_reader,
         wall_watch=wall_watch,
-        preprocessor=seq2seq_preprocessor,
-        postprocessor=lambda x, y: postprocesser(x, y, tokenizer),
+        preprocessor=evaluator_preprocessor,
+        postprocessor=lambda x, y: evaluator_postprocesser(x, y, tokenizer),
```

Comment:
let's use `functools.partial` for better readability

Reply:
I have done this in both `AsrDatasetConfig` and `whisper` PRs.